test_make_dir.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+def make_directory_test(tmpdir):
+    root = str(tmpdir)
+    target_dir = os.path.join(root, 'abcd')
+    generate_html_files(self, target_dir)
+    assert os.path.exists(target_dir) == 1
```

Comment:
One minor thing. this should be 
assert os.path.exists(target_dir)
as that method returns True/False